src/index.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
+// import './index.css';
```

Comment:
Great job @ijeomaodinko 🤝

- [ ] [Optional] Kindly remove commented code from your project as this is not regarded as a good practice.